### DISCUSSION TITLE
Is there an option to not hook certain functions in qiling?

### DISCUSSION THREAD

**Discussion by YKW1101** [qilingframework/qiling#1477] [Q&A]
I tried running a c program compiled with tdmgcc through qiling's windows_trace.py. 
```bash
python windows_trace.py -r -t target_program.exe
``` 
The program just outputs the string hello to puts. 
However, inside the _initterm function, it stores certain values in memory, which is not executed by qiling, so the puts were not executed.
```bash
unicorn.unicorn.UcError: Invalid memory write (UC_ERR_WRITE_UNMAPPED) 
```
This error is raised.

So I commented out the implementation part of the _intterm api in qiling\os\windows\dlls\msvcrt.py.
```python
@winsdkapi(cc=CDECL, params={
    'pfbegin' : POINTER,
    'pfend'   : POINTER
})
def hook__initterm(ql: Qiling, address: int, params):
    return 0
```
 After that, when I ran it again, it entered the original instruction of the _initterm api, stored the value in memory normally, and executed until puts. So I want to make the code execute the original instruction without hooking the execution of _initterm in the code instead of commenting out the dll code like this.
Is there an API that does this without hooking that is provided by qiling itself? I can implement an additional hooking function to execute it, but I would like to utilize the api of qiling.

**Comment by faniAhmed** [comment]
Did you find any solution?
I am also facing same issue
